introduction.md

```diff
@@ -0,0 +1 @@
+It is some text about me. I don't know what to tell about myself so I just wrote this text
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.